File: vbnet/TurnTicketDispenser.Tests/TicketDispenserTests.vb
Imports RacingCarKatas.TurnTicketDispenser
Imports Xunit

Public Class TicketDispenserTests
    <Fact>
    Public Sub Foo()
        Dim ticketDispenser = New TicketDispenser()
        Dim turnTicket = ticketDispenser.GetTurnTicket()
    End Sub
End Class

File: perl/test/TurnTicketDispenser/TicketDispenserTests.pl
use strict;
use warnings;

use Test::More 0.96;

use lib 'lib/TurnTicketDispenser';

require TicketDispenser;

subtest 'foo' => sub {
    my $ticketDispenser = new TicketDispenser();
    my $turnTicket = $ticketDispenser->getTurnTicket();
};

done_testing();